airflow/example_dags/example_kubernetes_executor_config.py

```diff
@@ -31,130 +30,146 @@
+            executor_config={
+                "pod_override": k8s.V1Pod(
+                    spec=k8s.V1PodSpec(
+                        containers=[
+                            k8s.V1Container(
+                                name="base",
+                                volume_mounts=[
+                                    k8s.V1VolumeMount(mount_path="/shared/", name="shared-empty-dir")
+                                ],
+                            ),
+                            k8s.V1Container(
+                                name="sidecar",
+                                image="ubuntu",
+                                args=["echo \"retrieved from mount\" > /shared/test.txt"],
+                                command=["bash", "-cx"],
+                                volume_mounts=[
+                                    k8s.V1VolumeMount(mount_path="/shared/", name="shared-empty-dir")
+                                ],
+                            ),
+                        ],
+                        volumes=[
+                            k8s.V1Volume(name="shared-empty-dir", empty_dir=k8s.V1EmptyDirVolumeSource()),
+                        ],
+                    )
+                ),
+            },
+        )
+        # [END task_with_sidecar]
 
-    # Test that we can add labels to pods
-    third_task = PythonOperator(
-        task_id="non_root_task",
-        python_callable=print_stuff,
-        executor_config={"pod_override": k8s.V1Pod(metadata=k8s.V1ObjectMeta(labels={"release": "stable"}))},
-    )
+        # Test that we can add labels to pods
+        third_task = PythonOperator(
+            task_id="non_root_task",
+            python_callable=print_stuff,
+            executor_config={
+                "pod_override": k8s.V1Pod(metadata=k8s.V1ObjectMeta(labels={"release": "stable"}))
+            },
+        )
 
-    other_ns_task = PythonOperator(
-        task_id="other_namespace_task",
-        python_callable=print_stuff,
-        executor_config={
-            "KubernetesExecutor": {"namespace": "test-namespace", "labels": {"release": "stable"}}
-        },
-    )
+        other_ns_task = PythonOperator(
+            task_id="other_namespace_task",
+            python_callable=print_stuff,
+            executor_config={
+                "KubernetesExecutor": {"namespace": "test-namespace", "labels": {"release": "stable"}}
+            },
+        )
 
-    start_task >> volume_task >> third_task
-    start_task >> other_ns_task
-    start_task >> sidecar_task
-    start_task >> task_with_template
+        start_task >> volume_task >> third_task
+        start_task >> other_ns_task
+        start_task >> sidecar_task
+        start_task >> task_with_template
+except ImportError as e:
+    log.warning("Could not import DAGs in example_kubernetes_executor_config.py: %s", str(e))
+    log.warning("Install kubernetes dependencies with: pip install apache-airflow['kubernetes']")
```

Comment:
Shall we use `cncf.kubernetes` instead? Given `kubernetes` is planned to be deprecated in 2.1 
https://github.com/sekikn/airflow/blob/41cf172d6af1a5406367e179df763ef1992e4108/setup.py#L604

Reply:
Yep. This is the official one. and it is in the docs.

Reply:
fixed in a73dcb846